x/tokenomics/keeper/keeper_test.go

```diff
@@ -62,7 +74,10 @@ func (s *TestSuite) SetupTest() {
 			SessionStartBlockHeight: 1,
 			SessionEndBlockHeight:   testsession.GetSessionEndHeightWithDefaultParams(1),
 		},
-		RootHash: smstRootWithSum(69),
+
+		// Set the suite expectedComputeUnits to be equal to the default threshold.
+		// This SHOULD make the claim require a proof given the default proof parameters.
```

Comment:
SHOULD or MUST? Should be the latter, no?

Reply:
SHOULD ... given the default parameters. The test cases may change the params to effect whether a proof is required or not. I was using SHOULD to describe the default behavior.